xpctl/README.md

```diff
@@ -1 +0,0 @@
-See the [docs](../docs/xpctl.md)
```

Comment:
ll this remove the README from the `xpctl` folder?

Reply:
Yeah, the `xpctl` folder is now only code but the `docs/xpctl.md` is linked as the README on pypi